### ISSUE TITLE
minimp3 uses an unsafe dependency

### ISSUE THREAD

**Issue by ImUrX** [RustAudio/rodio#456]
It uses slice-deque which has [RUSTSEC-2021-0047](https://rustsec.org/advisories/RUSTSEC-2021-0047) 
I saw that germangb/minimp3-rs#38 exists, it could be used instead of the current minimp3-rs.

**Comment by est31** [comment]
after #453 we now default to symphonia. I don't want to remove minimp3 completely as it still has its use cases, and I don't see alternatives that fulfill those use cases. Furthermore, it seems to me that the CVE is a hypothetical, that is, it requires a function to panic that is supplied by users. In fact, it seems that the `drain_filter` function isn't even used by minimp3 at all, so this is a false positive by tooling. Last, this is not a rodio issue. I'm closing in favour of https://github.com/germangb/minimp3-rs/issues/29 and most importantly https://github.com/gnzlbg/slice_deque/issues/90 .
